specs/mention_spec.js

```diff
@@ -0,0 +1,24 @@
+const assert = require('assert')
+const Botmock = require('botkit-mock')
+const mentionController = require('../features/mention.js')
+const responsesOnMention = require('../features/mention/replies')
+
+describe('Mentioning controller', () => {
+  beforeEach(() => {
+    this.controller = Botmock({})
+    this.bot = this.controller.spawn({ type: 'slack' })
+    mentionController(this.controller)
+  })
+
+  it('should return one of mention responds if user mentions bot', () => this.bot.usersInput([
+    {
```

Comment:
@nikitasakov I'd like to reduce amount of lines for brackets, because current style makes code less readable for multiple test cases